webpack.config.js

```diff
@@ -94,6 +94,9 @@ const webViewConfig = {
           {
             loader: "css-loader",
           },
+          {
+            loader: "resolve-url-loader",
```

Comment:
Question: Why this one is required?

Reply:
Because vscode codicons font is used in codicon.css with a relative path (./codicon.tff) to the css container directory. This loader helps to resolve css url(). I've noticed that when the extension is bundled into vsix, codicons were not rendering properly on Ubuntu.